tests/test_server.py

```diff
@@ -16,11 +17,19 @@
 print(x)
 """
 
+VERSION_REQUIREMENT_ASTRAL_DOCS = Version("0.0.291")
+
 
-class TestServer(unittest.TestCase):
+class TestServer:
```

Comment:
I need the pytest fixture to be accessible here

Reply:
I'd like to follow-up with a pull request that removes the class, but want to keep the diff focused here.